init/install_services.sh

```diff
@@ -32,7 +32,7 @@ kubectl create secret generic lmos-runtime --from-literal=OPENAI_API_KEY="$OPENA
 
 # Install lmos-runtime chart
 helm upgrade --install lmos-runtime oci://ghcr.io/lmos-ai/lmos-runtime-chart \
- --version 0.6.0 \
+ --version 0.0.11-SNAPSHOT \
```

Comment:
from a 0.6.0 release to a 0.0.11-SNAPSHOT?

Reply:
we had multiple snapshot versions from 0.6 to 0.11 but not released
0.0.11-SNAPSHOT has temp fix for CORS, do not want to enable it by default

Reply:
So I would assume you mean a `0.11.0-SNAPHOT` version instead of a `0.0.11-SNAPSHOT`?